### ISSUE TITLE
How to prepare doc translations for Vue 3 release?

### ISSUE THREAD

**Issue by ErickPetru** [vuejs/vuejs.org#2571]
Since I'm the main maintainer of the Brazilian Portuguese translation, I'm a bit worried... Based on @NataliaTepluhina [comment here](https://github.com/vuejs/vuejs.org/issues/2481#issuecomment-586700457), I suppose that the Vue 3 documentation is being reworked from scratch.

Did I understood it right? If so, is there anything our community could do to anticipate the work and be ready to release the new translation aligned (at least almost) with the Vue 3 release when it finally arrives?

**Comment by NataliaTepluhina** [comment]
@ErickPetru it's reworked from scratch indeed and we plan to publish it as soon as it's 'ready enough' for the translations. This should happen before the release, hopefully, next weeks. Why not right now? There are many moving parts and if translations will be started right now, it will add a lot of work on translators to rewrite things that already have been translated.

**Comment by awkj** [comment]
> @ErickPetru it's reworked from scratch indeed and we plan to publish it as soon as it's 'ready enough' for the translations. This should happen before the release, hopefully, next weeks. Why not right now? There are many moving parts and if translations will be started right now, it will add a lot of work on translators to rewrite things that already have been translated.

It's been 16 days, Any progress? looking forward to seeing it

**Comment by NatanCieplinski** [comment]
@NataliaTepluhina Any update on the status of the doc?

**Comment by varHarrie** [comment]
@NatanCieplinski Official docs have been published! https://github.com/vuejs/docs-next

**Comment by ErickPetru** [comment]
Yes @varHarrie, the repo is there, but not officially ready to release. @NataliaTepluhina is there any information about starting translations based in this new repo? Should we start it right now?

**Comment by bencodezen** [comment]
Hey @ErickPetru and @NatanCieplinski! We have just posted an official translations page on the v3 guide: https://v3.vuejs.org/guide/contributing/translations.html#can-we-start-translating-vue-3-docs

At this time, it is probably worth getting familiar with the codebase and how it works, but as far as major translation work, it might be best to still wait for the release candidate of the docs since we're still modifying things as we gather feedback.

Let me know if you have any other questions!
